src/index.ts

```diff
@@ -99,6 +99,9 @@ function getRepoDependencies(repo: BranchManifest) {
 	const depGraphManifests = repo.dependencyGraphManifests
 	const files = depGraphManifests.edges
 	let index = 0
+
+	const repoUpdateTime = repo.updatedAt
```

Comment:
Why updatedAt instead of pushedAt!!

https://github.com/orgs/community/discussions/24442